installers-conda/resources/post-install.sh

```diff
@@ -1,37 +1,65 @@
 #!/bin/bash
 set -e
 
-echo "*** Starting post install script for __NAME__.app"
-
-cat <<EOF
-* __PKG_NAME_LOWER__
-* __NAME__
-* __VERSION__
-* __CHANNELS__
-* __WRITE_CONDARC__
-* __SHORTCUTS__
-* __DEFAULT_PREFIX__
-* __LICENSE__
-* __FIRST_PAYLOAD_SIZE__
-* __SECOND_PAYLOAD_SIZE__
-* __MD5__
-* __INSTALL_COMMANDS__
-* __PLAT__
-* __NAME_LOWER__
-EOF
+echo "*** Running post install script for ${INSTALLER_NAME} ..."
 
 echo "Args = $@"
 echo "$(declare -p)"
 
-ENV_PREFIX=$(cd "${PREFIX}/envs/__NAME_LOWER__"; pwd)
-app_path="$(dirname ${DSTROOT})/Applications/__NAME__.app"
+name_lower=${INSTALLER_NAME,,}
+_shortcut_path="$HOME/.local/share/applications/${name_lower}_${name_lower}.desktop"
+shortcut_path="$(dirname ${_shortcut_path})/${name_lower}.desktop"
+if [[ -e ${_shortcut_path} ]]; then
+    echo "Renaming ${_shortcut_path}..."
+    mv -f "${_shortcut_path}" "${shortcut_path}"
+else
+    echo "${_shortcut_path} does not exist"
+fi
+
+case $SHELL in
+    (*"zsh") shell_init=$HOME/.zshrc ;;
+    (*"bash") shell_init=$HOME/.bashrc ;;
+esac
+spyder_exe=$(echo ${PREFIX}/envs/*/bin/spyder)
 
-if [[ -e "$app_path" ]]; then
-    echo "Creating python symbolic link..."
-    ln -sf "${ENV_PREFIX}/bin/python" "$app_path/Contents/MacOS/python"
+
+if [[ ! -e "$spyder_exe" ]]; then
+    echo "$spyder_exe not found. Alias not created."
+elif [[ -z "$shell_init" ]]; then
+    echo "Aliasing for $SHELL not implemented."
 else
-    echo "ERROR: $app_path does not exist"
-    exit 1
+    echo "Aliasing Spyder's executable in $shell_init ..."
+    sed -i "/alias spyder=/{h;s|=.*|=${spyder_exe}|};\${x;/^$/{s||\nalias spyder=${spyder_exe}|;H};x}" $shell_init
 fi
 
-echo "*** Post install script for __NAME__.app complete"
+echo "Creating uninstall script..."
+cat <<EOF > ${PREFIX}/uninstall.sh
+#!/bin/bash
+rm -rf ${shortcut_path}
+rm -rf ${PREFIX}
+EOF
+chmod +x ${PREFIX}/uninstall.sh
+
+cat <<EOF
+
+###############################################################################
+Spyder can be launched by standard methods in Gnome and KDE desktop
+environments. Additionally, Spyder can be launched in Gnome desktop
+environments from the command line:
+
+$ gtk-launch spyder
+
+Spyder can be launched from the command line for all Linux variants
+by:
+
+$ spyder
+
+To uninstall Spyder, from the command line:
+
+$ ${PREFIX}/uninstall.sh
```

Comment:
I think it'd be nice to add an `uninstall-spyder` alias to bashrc for this command and list it here instead. That would avoid people to remember what's the exact command to do it (which seems not so easy).

Reply:
Uninstallation should also remove the Spyder aliases in bashrc, but that could be left for later.